modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -122,16 +122,33 @@ definition {
 	test GuestConfigurationCanPersistInBrowserLocalStorage {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("Given guest user") {
```

Comment:
I think we need to add another step to toggle an option on, then toggle off, then check on the other tab, what do you think?